docs/messaging/telemetry.md

```diff
@@ -0,0 +1,33 @@
+---
+id: telemetry
+title: Telemetry
+slug: /messaging/telemetry
+---
+
+Event telemetry is sent for any interaction with the messaging surfaces and additionally for impressions (when the message is shown). The following types of user interaction is captured by telemetry:
+- Impression
+- Primary click
+- Secondary click (when available)
+- Dismiss
+
+Telemetry pings will include the message id, experiment and branch slug (if enrolled) and the event type.
+
+Telemetry from all messaging surfaces ([except doorhangers when not enrolled in an experiment](https://searchfox.org/mozilla-central/rev/65d4d3399afa79c8de5a0cc11752d2ba7c31edc1/browser/components/newtab/lib/TelemetryFeed.jsm#645)) includes client id in the event ping. **Special care should be taken not to associate any browsing history with messaging events: showing a message when the user visits a particular website is not allowed because it would accidentally leak information about that visit.**
+
+Additional telemetry is available when the message is delivered as an <u>experiment</u>:
```

Comment:
will be good to cross link telemetry documentation https://experimenter.info/telemetry/ available for experiments